coalib/settings/FunctionMetadata.py

```diff
@@ -91,6 +91,21 @@ def optional_params(self):
         """
         return self._filter_out_omitted(self._optional_params)
 
+    def add_alias(self, original, alias):
+        """
+        Adds an alias for the original setting. The alias setting will have
+        the same metadata as the original one. If the original setting is not
+        optional, the alias will default to ``None``.
+
+        :param original:  The name of the original setting.
+        :param alias:     The name of the alias for the original.
```

Comment:
does *the old setting name* stand for this?
if so i don't think it is clear enough :sweat_smile: